File: security/src/test/java/top/bettercode/summer/security/SecurityMvcControllerTest.java
package top.bettercode.summer.security;

import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;

import org.junit.jupiter.api.Test;
import top.bettercode.summer.test.BaseWebNoAuthTest;

public class SecurityMvcControllerTest extends BaseWebNoAuthTest {

  @Test
  public void test() throws Exception {
    mockMvc.perform(get("/testDefaultAuth")
    ).andExpect(status().isOk());
  }
}

File: security/src/test/kotlin/top/bettercode/summer/security/SecurityMvcControllerTest.kt
package top.bettercode.summer.security

import org.junit.jupiter.api.Test
import org.springframework.test.web.servlet.result.MockMvcResultMatchers
import top.bettercode.summer.test.BaseWebNoAuthTest

class SecurityMvcControllerTest : BaseWebNoAuthTest() {
    @Test
    @Throws(Exception::class)
    fun test() {
        mockMvc.perform(get("/testDefaultAuth")
        ).andExpect(MockMvcResultMatchers.status().isOk())
    }
}
